eoscompanion/main.py

```diff
@@ -157,7 +157,7 @@ def do_activate(self):  # pylint: disable=arguments-differ
         logging.info('Activated')
         return Gio.Application.do_activate(self)
 
-CONFIG_FILE = '/etc/eos-companion-app/config.ini'
+CONFIG_FILE = '/var/run/host/etc/eos-companion-app/config.ini'
```

Comment:
Good that this works in the sandbox - though I think this still needs to traverse the config directories in priority order. You may also need to add `--filesystem=/var/lib/eos-companion-app`